meetings/util.py

```diff
@@ -0,0 +1,10 @@
+from mautrix.types import EventType
+from mautrix.errors import MNotFound
+
+async def get_room_alias(client, room_id):
+    try:
+        existing_event = await client.get_state_event(room_id, EventType.ROOM_CANONICAL_ALIAS)
+        return existing_event.canonical_alias
+    except MNotFound:
+        # typically if a room is a direct message, it wont have a canonical alias
+        return None
```

Comment:
Since we *always* use the output of this function in the messages, should we return a useful string here? Maybe a shortened form of the room ID or similar?